### PR TITLE
add logging information for survival curve endpoint call

### PR DESCRIPTION
**Pull request by grugna**
Description about what this pull request does.

Please make sure to follow the [DEV guidelines](https://gen3.org/resources/developer/dev-introduction/) before asking for review.

### New Features
- Implemented XXX

### CODE REVIEW

**Comment by grugna** [comment]
@bobaekang I just extracted all the attributes from the payload at the beginning rather than all around since they are used multiple times and the log needs them too. Could you please work on this branch to align the survival curve endpoint to the new ES mapping?

File: PcdcAnalysisTools/blueprint/routes/views/survival/__init__.py [hunk 1]
```diff
@@ -16,31 +16,47 @@
...
-    )
-    return flask.jsonify(get_survival_result(data, args))
-
 
-def fetch_data(args):
     # TODO add json payload control
     # TODO add check on payload nulls and stuff
     # TODO add path in the config file or ENV variable
     _filter = args.get("filter")
+    filters = json.loads(json.dumps(_filter))
```
Comment (bobaekang) [inline PcdcAnalysisTools/blueprint/routes/views/survival/__init__.py:24]: @grugna Why do we need to dump/load JSON here? Isn't this effectively getting back the same object?

**Comment by bobaekang** [comment]
@grugna Why don't I create a new PR focused on that instead of putting both logging and ES mapping in a single PR?

**Comment by grugna** [comment]
Not sure, you did the loads/ dumps in series, I am sure there was a reason
for it.

Luca Graglia
312-918-4523
***@***.***

**Comment by grugna** [comment]
That would make more sense, I'll close this one then.

Luca Graglia
312-918-4523
***@***.***

**Comment by bobaekang** [comment]
> Not sure, you did the loads/ dumps in series, I am sure there was a reason for it.

Hmm, yeah, [I certainly added that line](https://github.com/chicagopcdc/PcdcAnalysisTools/pull/4/commits/8dc56839548d043b50292ad7e92a09b326ff7105) but can't really remember why. Probably the original input is not really a python dict and had to be converted as such?

I found [this SO answer](https://stackoverflow.com/a/32047677) where running `json.load(json.dump(x))` is suggested as a workaround for "Expecting property name enclosed in double quotes" error. Maybe that's why... I'll double check on this.

**Pull request merged**